modular_splurt/code/modules/mob/living/silicon/pai/pai.dm

```diff
@@ -21,3 +21,8 @@
 			has_penis = FALSE
 			has_balls = FALSE
 			has_vagina = FALSE
+		if("Toggle Breasts")
+			if(!has_breasts)
+				has_breasts = TRUE
+			else
+				has_breasts = FALSE
```

Comment:
You can replace the if and subsequent lines with `has_breasts = !has_breasts`

Reply:
I'll change that in the morning then!